### PR TITLE
DM-31801: Use lsst prefix for loggers

### PR DESCRIPTION
**Pull request by timj**
I think in order to switch to python logging we'll have to add a logging.basicConfig and lsst.log forwarding in here somewhere.

### CODE REVIEW

File: python/lsst/ap/verify/ingestion.py [hunk 1]
```diff
@@ -48,6 +48,8 @@
 from lsst.pipe.tasks.ingestCalibs import IngestCalibsTask
 from lsst.pipe.tasks.ingestCuratedCalibs import IngestCuratedCalibsTask
 
+_LOG = lsst.log.Log.getLogger(__name__)
```
Comment (kfindeisen) [inline python/lsst/ap/verify/ingestion.py:51]: Do we need this boilerplate in every module? I'm a bit suspicious of global variables...

File: python/lsst/ap/verify/ap_verify.py [hunk 2]
```diff
@@ -188,7 +190,7 @@ def runApVerify(cmdLine=None):
         or 127 if the task runner framework failed.
     """
     lsst.log.configure()
```
Comment (kfindeisen) [inline python/lsst/ap/verify/ap_verify.py:192]: I believe this call and the one in `runIngestion` are what should be replaced with any `logging.basicConfig` and `lsst.log` forwarding that is needed. These two functions should be the only (non-test) entry points to the code.

**Review by kfindeisen — APPROVED** [review]
I'm a bit unclear about the scope of this work, but have no serious objections to the changes.

File: python/lsst/ap/verify/ingestion.py (hunk 1 above)
Comment (timj) [reply python/lsst/ap/verify/ingestion.py:51]: Where else would you put it? We use this form all the time in middleware.

**Comment by timj** [comment]
@kfindeisen I switched it to python logging. I'm not sure what default log configuration you'd like but I use INFO and log messages to stdout.

**Comment by timj** [comment]
The scope of this work is to change the logger names but there is also a general requirement to switch all python code over to python logging where practical. Since I was already in the files I thought I may as well do that on this ticket.

File: python/lsst/ap/verify/ap_verify.py [hunk 3]
```diff
@@ -41,7 +43,17 @@
...
+
+
+def _configure_logger():
+    """Configure Python logging.
+
+    Does basic Python logging configuration and
+    forwards LSST logger to Python logging.
+    """
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+    lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
```
Comment (kfindeisen) [inline python/lsst/ap/verify/ap_verify.py:56]: No objection to `INFO`, but what is this last line?

Comment (kfindeisen) [reply python/lsst/ap/verify/ap_verify.py:56]: Never mind, it's obvious.

**Review by kfindeisen — APPROVED** [review]

File: python/lsst/ap/verify/ap_verify.py (hunk 3 above)
Comment (timj) [reply python/lsst/ap/verify/ap_verify.py:56]: That forwards all DEBUG and above messages to python logging -- python will drop the actual DEBUG messages. MDC is disabled.

**Pull request merged**